server/controllers/recipes.js

```diff
@@ -6,6 +6,13 @@ import validateUserRight from '../services/validateUserRight';
 import cloudinary, { uploadWithMulter } from '../services/uploadImage';
 import populatePaging from '../services/populatePaging';
 
+/**
+ * Check if recipe name is picked
+ *
+ * @param {any} userId
```

Comment:
param type `any` could be `integer`.  Use the proper param type, I see this in other places in your code